TeachingRecordSystem/src/TeachingRecordSystem.Api/V1/Handlers/GetTeacherHandler.cs

```diff
@@ -96,13 +96,21 @@ Induction MapInduction()
                 {
                     StartDate = induction.dfeta_StartDate,
                     CompletionDate = induction.dfeta_CompletionDate,
-                    InductionStatusName = induction.FormattedValues.ContainsKey(dfeta_induction.Fields.dfeta_InductionStatus) ?
-                        induction.FormattedValues[dfeta_induction.Fields.dfeta_InductionStatus] :
+                    InductionStatusName = teacher.FormattedValues.ContainsKey(Contact.Fields.dfeta_InductionStatus) ?
+                        teacher.FormattedValues[Contact.Fields.dfeta_InductionStatus] :
                         null,
                     State = induction.StateCode.Value,
                     StateName = induction.FormattedValues[dfeta_induction.Fields.StateCode]
                 } :
-                null;
+                teacher.dfeta_qtlsdate.HasValue ?
+                    new Induction()
+                    {
+                        StartDate = null,
+                        CompletionDate = null,
+                        InductionStatusName = teacher.FormattedValues.ContainsKey(Contact.Fields.dfeta_InductionStatus) ?
```

Comment:
This conditional should be moved up a level (i.e. return a `null` `Induction` if there's no status)